test/integration/jest.config.json

```diff
@@ -0,0 +1,16 @@
+{
+	"modulePaths": [
+		"<rootDir>/test/",
+		"<rootDir>/server/",
+		"<rootDir>/client/",
+		"<rootDir>/client/extensions/"
+	],
+	"rootDir": "./../../",
+	"testEnvironment": "node",
+	"testMatch": [
+		"**/bin/**/__integration__/*.js",
+		"**/client/**/__integration__/*.js",
+		"**/server/**/__integration__/*.js"
```

Comment:
Are there any technical hurdles for using `integration` as a match instead? I do ❤️ that these are inline now with the related src.